src/reference/asciidoc/amqp.adoc

```diff
@@ -1705,6 +1705,8 @@ new MessageListenerAdapter((ReplyingMessageListener<String, String>) data -> {
 ----
 ====
 
+Starting with version 2.2, we deprecate method `buildListenerArguments` and provide new one with complete arguments including `Channel` and `Message`, which helping listener to receive customized arguments, such as getting `Channel` to basicReject in manual acknowledge mode by means of doing extends `MessageListenerAdapter` and overriding `buildListenerArguments(Object, Channel, Message)` then return array carried `Channel` and `Message`.
```

Comment:
Please, be more specific which method we deprecate: `buildListenerArguments(Object)`.
"and provide a new one" ?

The `basicReject` must be wrapped into the code, too.

Reply:
changed